polynote-kernel/src/main/scala/polynote/kernel/dependency/CoursierFetcher.scala

```diff
@@ -65,8 +66,14 @@ object CoursierFetcher {
     } yield downloaded
   }
 
-  private def loadCredentials(credentials: CredentialsConfig): Task[List[DirectCredentials]] = credentials.coursier match {
-    case Some(CredentialsConfig.Coursier(path)) => Task(CoursierCredentials(new File(path)).get().toList)
+  private def loadCredentials(credentials: CredentialsConfig): URIO[Logging, List[DirectCredentials]] = credentials.coursier match {
+    case Some(CredentialsConfig.Coursier(path)) =>
+      Task(CoursierCredentials(new File(path), optional = false).get().toList)
+        .catchAllCause(cause => for {
```

Comment:
This could be `.catchAll(err => Logging.error("Failed to load credentials", err).as(Nil))`

Reply:
Thanks for the heads up. Fixed in 607b7c8.